### DISCUSSION TITLE
Prisma stuck in nuxt 3 application

### DISCUSSION THREAD

**Discussion by cawa-93** [prisma/prisma#13389] [Q&A] [answered]
I'm trying to integrate prisma into the nuxt 3 application. 
[Here is my reproduction.](https://github.com/cawa-93/nuxt-prisma-hmr-issue).

I Call Prisma in single server route:
https://github.com/cawa-93/nuxt-prisma-hmr-issue/blob/e42d72ca6fd6226601971f6001a9342764e9f45a/server/api/users.ts

For some reason, after a hot reboot of the server route, the prisma gets stuck. I guess it's not a bug on the prisma side. But I have no idea where to start looking for a reason. Perhaps experienced users of the prisma can suggest what could be the cause of such?

**Comment by nurul3101** [comment]
Hey @cawa-93 👋 

Do you get any errors? Any timeout error? Or just nothing happens?

**Reply by cawa-93** [reply]
No. No error at all. I have the impression that nuxt is stuck at the stage of reload modules. Maybe there are some listeners of events, or connections to the database that he can not break? When I tried debug, nuxt stopped at the Promise. The Promise note was neither rejected nor resolved. It just never finalized.

**Comment by chuahziyang** [comment]
Having this same issue, seems to be related to prisma dropping the connection on reload 
[](https://github.com/prisma/prisma/discussions/14666)

**Comment by g4w3l** [comment, marked as answer]
Got the same issue, have you found a fix for this ? Or at least, what to do to manually get Prisma unstuck ?

**Reply by cawa-93** [reply]
No. This issue forced me work on project in WSL

**Reply by g4w3l** [reply]
Thanks, I will try this, if it works on WSL, it's still a workaround 😅 !

![image](https://user-images.githubusercontent.com/621235/190829066-3bbc151a-cc68-440f-90ba-f4e378ab3d38.png)
